tests/dotnet/UnitTests/TemplateTest.cs

```diff
@@ -191,18 +247,20 @@ public void CreateAndBuildItemTemplates (ApplePlatform platform)
 			// item templates for the given platforms. This is faster than testing the item templates
 			// one-by-one.
 
-			var info = Templates.Single (v => string.Equals (v.Template, platform.AsString (), StringComparison.OrdinalIgnoreCase));
```

Comment:
I think the `CreateAndBuildProjectTemplate` test needs to be updated to pass the language when instantiating the template, otherwise it'll just create the C# version of the `ios` template twice (since that's the default language).

Reply:
If I'm not mistaking, I changed this specific test to run only for C# by making sure to filter the list of templates (`language is null or csharp`) to pass the language flag every time we call the `DotNet` command line.

I have put F# into a separate test (`CreateAndBuildFSharpProjectTemplate`).

Do you see somewhere where I forgot to pass the language flag?
If not, maybe just renaming this test to `CreateAndBuildCSharpProjectTemplate` would be better?

Reply:
I think I confused you, I'm talking about the `CreateAndBuildProjectTemplate` test, but I added the comment to changes you made to the `CreateAndBuildItemTemplates` test.

You didn't make any changes to the `CreateAndBuildProjectTemplate` test, and this is where the language flag is missing:

https://github.com/TimLariviere/xamarin-macios/blob/ce983c67737e8594dbd05f6175e256228b907bec/tests/dotnet/UnitTests/TemplateTest.cs#L200

nor can I find that you've filtered to C# projects only.

Reply:
Oh ok, I see now. I'll fix that.

Reply:
@rolfbjarne I updated the tests to use the language identifier everytime